airbyte-integrations/connectors/source-unleash/source_unleash/unleash.yaml

```diff
@@ -0,0 +1,55 @@
+version: "0.1.0"
+
+definitions:
+  selector:
+    extractor:
+      field_pointer: []
+  selector_features:
+    extractor:
+      field_pointer: ["features"]
+  requester:
+    url_base: "{{ config['api_url'] }}/api/client"
+    http_method: "GET"
+    request_options_provider:
+      request_headers:
+        "Content-Type": "application/json"
+        "Authorization": "{{ config['api_token'] }}"
+
+    error_handler:
+      type: "DefaultErrorHandler"
+      error_handlers:
+        - response_filters:
+            - http_codes: [401]
+              action: FAIL
+            - http_codes: [403]
+              action: RETRY
+  retriever:
+    record_selector:
+      $ref: "*ref(definitions.selector)"
+    paginator:
+      type: NoPagination
```

Comment:
The API doesn't have pagination?

Reply:
I've scanned the whole API documentation and couldn't find a single line on the pagination logic 😢 so I suspect that it doesn't